db/createItem_test.go

```diff
@@ -14,6 +14,7 @@ func TestCreateItem(t *testing.T) {
 	listID := "474c2Fff7"
 	itemID := "b6cf642d"
 	itemName := "Peaches"
+	isCompleted := false
```

Comment:
Could be using `boolToPointer` and `stringToPointer` at the point of the test case definitions, instead of having these declared up here.
Optional

Reply:
(I'm not sure where we wrote those functions - possible they are in different packages)